concepts/framework/flow-concept.md

```diff
@@ -18,6 +18,18 @@ This is a business rule to determine whether the action should be executed.
 
 The tasks that execute when the trigger happens or the certain conditions are met. There is a special action called "Stop flow", which will stop any further action below.
 
+## Flow Templates
+
+Flow Templates are flows which have been built that have been designated as a template.
+The Flow library contains flows, which are shipped with Shopware, Apps or Plugins. There are two main ways to create a flow template in the template library:  
+
+* Flow templates are created by Apps.
+* Flows templates are created by Plugins.
+
+We can help merchants to reduce complexity when building a flow by enabling them to create an automation process in their business. As a merchant, you may design a Flow more easily by using the Flow Templates. So you don't have to create complicated flows on your own.
+
+You can view the details of a flow template just like a regular flow. However, Flow templates cannot be modified.
+
```

Comment:
```suggestion
## Flow Templates
		
Pre-created [flows](#flow) built are designated as flow templates.

The flow library contains flows that are shipped with Shopware. Two main ways to create a flow template in the template library are by [apps](../../guides/plugins/plugins/framework/flow/README.md) and [plugins](../../guides/plugins/apps/flow-builder/README.md).

We can help merchants reduce the complexity of creating an automation process in their business by using a flow template rather than building a flow. As a merchant, you may design a flow more easily by using the flow templates. So you don't have to create complicated flows on your own.		

You can view the details of a flow template just like a regular flow. However, flow templates can't be modified.
```

Reply:
Thanks for your suggestions @sushmangupta. I've updated this doc. Could you review it again?